__tests__/app.test.js

```diff
@@ -117,6 +117,29 @@ describe("GET /api/articles",() => {
             expect(body.message).toBe('Invalid query')
         })
     })
+    test("GET: 200 responds with the article objects filtered by topic query",() => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles.length).toBeGreaterThan(0)
+            expect(body.articles[0]).toMatchObject({
+                author: 'icellusedkars',
+                title: 'Eight pug gifs that remind me of mitch',
+                article_id: 3,
+                topic: 'mitch',
+            })
+        })
+    })
```

Comment:
It might be a nice idea to check that all of the articles have a topic of 'mitch'? :)